### DISCUSSION TITLE
collection/iterator-iterable

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#149] [Announcements]
# collection/iterator-iterable

Java程序员进阶之路，小白的零基础Java教程，Java中Iterator和Iterable有什么区别？

https://tobebetterjavaer.com/collection/iterator-iterable.html

**Comment by 2258324319** [comment]
非常有用了可以说，请维护到我退休

**Reply by itwanger** [reply]
嗯，已经在优化第一版了。

**Comment by SkMoran** [comment]
二哥写的确实清晰明了 我个人感觉比java guide 什么的还要好 加油！！
